spec/rubocop/formatter/formatter_set_spec.rb

```diff
@@ -83,6 +83,16 @@ module Formatter
           formatter_set.add_formatter('simple')
         end
 
+        around do |example|
+          begin
+            stdout = $stdout
+            $stdout = StringIO.new
+            example.run
+          ensure
+            $stdout = stdout
+          end
+        end
```

Comment:
With test-queue, `$stdout` may be a file, so, `$stdout` was closed in this test.
So, I changed it that replace `$stdout` with dummy IO.